src/main/java/org/codehaus/mojo/versions/xml/PluginUpdatesXmlRenderer.java

```diff
@@ -9,7 +9,7 @@
  * "License"); you may not use this file except in compliance
  * with the License.  You may obtain a copy of the License at
  *
- *     http://www.apache.org/licenses/LICENSE-2.0
+ *     https://www.apache.org/licenses/LICENSE-2.0
```

Comment:
not related change, 
by the way license headers for Mojo House project should be discussed on dev list.
I'm not sure if should be ... `Licensed to the Apache Software Foundation (ASF)`

I would like to not touch this now.